docs/design-decisions/README.md

```diff
@@ -5,26 +5,64 @@
 ## Outstanding Questions
 
 - Q: How does the switch _know_ to send a callback to the PISP after a sucessful transfer?
-  - when DFSPA issues `POST /transfer`, that's between DFSPA  + DFSPB
-  - It should have a `transferId` that is related to the `thirdPartyRequest`
-    - how does the PISP or Switch know which transaction this `transferId` is related?
-    - tx id will be set in the `POST /authorizations` (when the authorization is created by the DFSP)
-    - how do we get from a `transactionId` to a `transferId`?
-      - ILP packet is in transfer request, but encoded... so need to figure this piece out
-    - For now, assume that the `Transaction` object will contain the `transferId`
-  > Note: this is being tracked in [#270](https://app.zenhub.com/workspaces/pisp-5e8457b05580fb04a7fd4878/issues/mojaloop/mojaloop/270)
+    - Refer to [#42](https://github.com/mojaloop/pisp/issues/42)
 
-- Q. How does the switch determine whether or not a DFSP is using their own FIDO service? Do we want to use the ALS or some other method?
+- Q: Will we make a new `thirdparty-scheme-adapter` to handle thirdparty requests?
+    - Signs point to yes at the moment, but the challenge is how to divide between the existing `sdk-scheme-adapter` and a new `thirdparty-scheme-adapter`
 
-- Q. Which api should the DFSP need to implement for PISP functionaliy? We have a few options:
-    1. Add the DFSP changes to the existing FSPIOP-API
-    2. Add the DFSP changes to the new `thirdparty-api`
-    3. Divide the `thirdparty-api` into 2 parts: 
-        - `thirdparty-pisp-api` for the PISP to implement
-        - `thirdparty-dfsp-api` for the DFSP to implement
+## Decisions Made
 
+### How does the switch determine whether or not a DFSP is using their own FIDO service? Do we want to use the ALS or some other method?
+
+We will use the ALS to record the auth service for a given participant
```

Comment:
so once its established that a DFSP is using its own auth service does the switch still route the request onwards to the DFSP or does does the auth request bypass the switch?

Reply:
The switch will just send the request on to the DFSP, and the DFSP can worry about it from there.

I've actually been doing a little more thinking on this, and perhaps we don't even need to use the ALS, but instead register plain old callbacks like we're doing with all of our other services.

Ref: https://github.com/mojaloop/sdk-standard-components/pull/71